js/number/Index.svelte

```diff
@@ -31,6 +31,10 @@
 	export let interactive: boolean;
 	export let placeholder = "";
 
+	if (value === null && placeholder === "") {
```

Comment:
why is `placeholder === ""` check necessary?

Reply:
we only set the value to none if there’s no placeholder, otherwise the placeholder would never show